docs/source/whatsnew/1.0.0b2.rst

```diff
@@ -1,6 +1,6 @@
 .. _whatsnew_100b2:
 
-1.0.0b2 (???)
+1.0.0b2 (November 15, 2019)
```

Comment:
I think the --- below is supposed to have the same number of characters as this line